src/overview/build-deploy.md

```diff
@@ -9,36 +9,36 @@ Every time you push to a live branch (a git branch with an active environment at
 
-That build process is under your control, and reruns on every Git push.  Every Git push is a validation not only of your code but of your build process.  Whether that's installing packages using Composer or Bundler, compiling TypeScript or Sass, or building your application code in Go or Java, your build process is vetted every time you push.
+That build process is under your control and runs on every Git push.  Every Git push is a validation not only of your code, but of your build process.  Whether that's installing packages using Composer or Bundler, compiling TypeScript or Sass, or building your application code in Go or Java, your build process is vetted every time you push.
 
-Whenever possible you should avoid committing build assets to your repository that can be regenerated at build time.  Depending on your application that may include 3rd party libraries and frameworks, generated and optimized CSS and JS files, generated source code, etc.
+Whenever possible, you should avoid committing build assets to your repository that can be regenerated at build time.  Depending on your application, that may include 3rd party libraries and frameworks, generated and optimized CSS and JS files, generated source code, etc.
 
-The following two constraints make sure you have, fast, repeatable builds:
+The following two constraints make sure you have fast, repeatable builds:
 
-1. The build step should be environment independent (this is paramount to ensure development environments are truly perfect copies of production). This means you can not connect to services (like the database) during the build.
-2. The resulting built application must be Read-Only, if your application requires writing to the file-system,  this can easily be done by specifying which are the directories that require Read/Write access. These should not be directories that have code because that would be a security risk.
+1. **The build step should be environment-independent.** This is paramount to ensure development environments are truly perfect copies of production. This means you can not connect to services (like the database) during the build.
+2. **The final built application must be read-only.** If your application requires writing to the filesystem, you can specify the directories that require Read/Write access. These should not be directories that have code, because that would be a security risk.
 
 ## Building the application
 
-After you push your code the first build step is to validate your configuration files (`.platform.app.yaml`, `.platform/services.yaml` and `.platform/routes.yaml`). The git server will issue an error if validation fails, and nothing will have happened on the server.
+After you push your code, the first build step is to validate your configuration files (i.e. `.platform.app.yaml`, `.platform/services.yaml`, and `.platform/routes.yaml`). The Git server will issue an error if validation fails, and nothing will happen on the server.
```

Comment:
First comma is extraneous.

Reply:
Introductory clauses should always have a comma. Basically, if you can invert the sentence, it should have a comma.

Thus, "The first build step is to validate your configuration files after you push your code" requires a comma when it's inverted to "After you push your code, the first build step..."

Reply:
Stet.